cypress/integration/group2/organizations.ts

```diff
@@ -306,7 +306,7 @@ describe('Dockstore Organizations', () => {
     });
 
     // test the fix for DOCK-1945
-    it('stay on collections page when removing an entry',  () => {
+    it('stay on collections page when removing an entry', () => {
```

Comment:
@svonworl These and a lot of the diffs that Kim didn't change seem to be yours.

The husky [pre-commit hook](https://github.com/dockstore/dockstore-ui2/blob/01ed8b6080ec7afd2ab6786df5fe7b20c734a3d1/.husky/pre-commit#L4) is what's doing this. Any idea why it's not happening for your commits, if that indeed is the issue?

Reply:
Yes, it appears to be mangling my well-formatted code :/ (with the exception of deleting the extra space above, which was merited :)   Hmm, maybe it got disabled somehow, will check.  Tough question: do we really like what it does to the HTML, in general?

Reply:
Think we're using prettier https://github.com/prettier/prettier 
IIRC, its pretty opinionated though you could try upgrading it to see if it does anything in edge cases.

Reply:
1. I think it's good to be standardized so that PRs like this don't get confusing, so reviewers can focus on the PR proper.
2. I don't always like all the formatting choices either. They're probably configurable via prettier if you want to submit PRs if you have strong opinions on particular formatting.
3. I'm also open to other ideas, but the idea of doing it this way is so non-standardized commits don't slip through. Although apparently your code did, which is confusing. But if you can figure out a different way to ensure it happens reliably. ok.